swatch-product-configuration/src/main/resources/subscription_configs/RHEL/RHEL_for_x86_ha.yaml

```diff
@@ -0,0 +1,15 @@
+---
+platform: RHEL
+
+id: rhel-for-x86-ha
+
```

Comment:
I'm not sure if https://github.com/RedHatInsights/rhsm-subscriptions/blob/4a73665a5d8c32bc5169751d30213d5593a62410/swatch-product-configuration/src/main/resources/subscription_configs/RHEL/RHEL_for_x86_EUS.yaml#L5 should also be added here. Any hints?

Reply:
@Sgitario nope, we don't need to add it there.  RHEL HA is standalone, and we shouldn't count it as including rhel for x86.  Same is true for RHEL RS (SWATCH-1523) configuration.